### PR TITLE
OCP 4.15 Installation doc updates for IBM Power

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.15

Issue: https://issues.redhat.com/browse/MULTIARCH-3998

Link to docs preview:
IBM Power : https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/preparing-to-install-on-ibm-power
- https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#installation-bare-metal-config-yaml_installing-ibm-power
- https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#installation-load-balancing-user-infra-example_installing-ibm-power
- https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#installation-user-infra-machines-iso_installing-ibm-power
- https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#nw-operator-cr-cno-object_installing-ibm-power
- https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#recommended-ibm-power-system-requirements_installing-ibm-power

QE review:
- [x] QE has approved this change.

Additional information:
Review doc: https://docs.google.com/document/d/1-eL_iI0_qy-jYBGRtOPSWclDuj5KtIw7y0C7FJ_DWdk/edit

### CODE REVIEW

**Comment by manojnkumar** [comment]
The Sample API and Ingress section still has master/worker terminology:

```
listen machine-config-server-22623 
  bind *:22623
  mode tcp
  option  httpchk GET /readyz HTTP/1.0
  option  log-health-checks
  balance roundrobin
  server bootstrap bootstrap.ocp4.example.com:6443 verify none check check-ssl inter 10s fall 2 rise 3 backup 
  server master0 master0.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
  server master1 master1.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
  server master2 master2.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
listen ingress-router-443 
  bind *:443
  mode tcp
  balance source
  server worker0 worker0.ocp4.example.com:443 check inter 1s
  server worker1 worker1.ocp4.example.com:443 check inter 1s
  ```

File: modules/cli-installing-cli.adoc [hunk 1]
```diff
@@ -210,7 +210,7 @@ ifndef::openshift-origin,microshift[]
 +
 [NOTE]
 ====
-For macOS arm64, choose the *OpenShift v{product-version} macOS arm64 Client* entry.
+For macOS arm64, choose the *OpenShift v{product-version} macOS aarch64 Client* entry.
```
Comment (alishaIBM) [inline modules/cli-installing-cli.adoc:213]: @juliemathew @pravin-dsilva request you to please provide your inputs on above change. Thanks.

**Comment by alishaIBM** [comment]
> The Sample API and Ingress section still has master/worker terminology:
> 
> ```
> listen machine-config-server-22623 
>   bind *:22623
>   mode tcp
>   option  httpchk GET /readyz HTTP/1.0
>   option  log-health-checks
>   balance roundrobin
>   server bootstrap bootstrap.ocp4.example.com:6443 verify none check check-ssl inter 10s fall 2 rise 3 backup 
>   server master0 master0.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
>   server master1 master1.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
>   server master2 master2.ocp4.example.com:6443 weight 1 verify none check check-ssl inter 10s fall 2 rise 3
> listen ingress-router-443 
>   bind *:443
>   mode tcp
>   balance source
>   server worker0 worker0.ocp4.example.com:443 check inter 1s
>   server worker1 worker1.ocp4.example.com:443 check inter 1s
> ```

@manojnkumar I have updated the `installation-load-balancing-user-infra` file. Thanks.

**Comment by manojnkumar** [comment]
LGTM

File: modules/cli-installing-cli.adoc (hunk 1 above)
Comment (kelbrown20) [reply modules/cli-installing-cli.adoc:213]: If this was for linux, I think this change would be correct. However, apple and macOS only uses the `arm64` format. So I believe it should just be `macOS arm64 Client`

Comment (alishaIBM) [reply modules/cli-installing-cli.adoc]: @kelbrown20 thank you for the clarification. I have updated the PR as per above. Thanks.

cc: @manojnkumar @juliemathew @pravin-dsilva

File: modules/nw-operator-cr.adoc [hunk 2]
```diff
@@ -347,6 +347,10 @@ defaultNetwork:
     genevePort: 6081
     ipsecConfig: {}
 ----
+[IMPORTANT]
+====
+Using `OVNKubernetes` can lead to stack exhaustion problem on {ibm-power-name}.
```
Comment (mramendi) [inline modules/nw-operator-cr.adoc:352]: I suggest changing to "stack exhaustion problems" or "a stack exhaustion problem" as appropriate, for the sake of grammar.

**Review by mramendi — COMMENTED** [review]
Only one grammar issue. 

Note also that I was unable to find most of the text at the preview link. Given the limited extent of the changes this is not a concern for me. However, for the merge review I would suggest adding a complete set of preview links.

As I an on peer review induction @JoeAldinger will finish the review

**Comment by JoeAldinger** [comment]
Please add direct links to the PR @alishaIBM this slows reviewers down when we have to go looking for them.

**Comment by JoeAldinger** [comment]
Links for the first conditional statement: [IBM](https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#installing-ibm-power:~:text=pool%20is%20used.-,Simultaneous,-multithreading%20(SMT)%20is) and [IBM-z](https://70553--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z#:~:text=Specifies%20whether%20to,and%20compute%20machines.)

File: modules/installation-user-infra-machines-static-network.adoc [hunk 3]
```diff
@@ -293,7 +293,12 @@ Optional: You can bond multiple SR-IOV network interfaces to a dual port NIC int
 
 On each node, you must perform the following tasks:
 
+ifndef::installing-ibm-power[]
 . Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in the "Attaching SR-IOV networking devices to virtual machines" section.
+endif::installing-ibm-power[]
```
Comment (JoeAldinger) [inline modules/installation-user-infra-machines-static-network.adoc:298]: I'm not an expert on conditionals but are lines 298 and 299 necessary? Couldn't you open the conditional at line 296 and close it at line 301?

File: modules/nw-operator-cr.adoc (hunk 2 above)
Comment (JoeAldinger) [inline modules/nw-operator-cr.adoc:352]:
```suggestion
Using OVN-Kubernetes can lead to stack exhaustion problem on {ibm-power-name}.
```

Comment (JoeAldinger) [reply modules/nw-operator-cr.adoc:352]: I've seen the backticks for when it is a literal value, but never for a general reference. However, I then started looking around and saw that most of the install docs use the backticks so I'm not sure. I write for OVN-K and now you have me wondering if we need to address this in the style guide.

**Review by JoeAldinger — COMMENTED** [review]
Few suggestion and one question.

File: modules/nw-operator-cr.adoc (hunk 2 above)
Comment (alishaIBM) [reply modules/nw-operator-cr.adoc]: @JoeAldinger Thank you the suggestion. I agree. Updated the PR accordingly.

File: modules/installation-user-infra-machines-static-network.adoc (hunk 3 above)
Comment (alishaIBM) [reply modules/installation-user-infra-machines-static-network.adoc:298]: @JoeAldinger updated the code as below:
```
ifndef::installing-ibm-power[]
. Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in the "Attaching SR-IOV networking devices to virtual machines" section.
endif::installing-ibm-power[]
ifdef::installing-ibm-power[]
. Create the SR-IOV virtual functions (VFs).
endif::installing-ibm-power[]
```
Thank you for pointing out. The second part is power only.

File: modules/nw-operator-cr.adoc (hunk 2 above)
Comment (alishaIBM) [reply modules/nw-operator-cr.adoc]: Thanks @mramendi 
Updated as: "a stack exhaustion problem".

**Comment by alishaIBM** [comment]
> Please add direct links to the PR @alishaIBM this slows reviewers down when we have to go looking for them.

Updated the links. Thanks.

**Pull request merged**